doc/en/api/1.0.0/importer.yaml

```diff
@@ -99,14 +127,585 @@ paths:
+    type: boolean
+    required: false
+  ExpandParameter:
+    name: expand
+    type: string
+    in: path
+    description: 'What level to expand the response object to. Can be "self" (expand only the response object and its immediate children), "all" (expand all children), "none" (don''t include any children), or a nonnegative integer, indicating the depth of children to expand to. Defaults to "self" or "none", depending on the request.'
+    required: false
+  ContextParameter:
+    name: importBody
+    description: The import context to create.
+    in: body
+    required: true          
+    schema:
+      $ref: "#/definitions/Context"
 definitions:
-  Produces:
-    - application/json
-    - text/html
-  PostParameters:
-    - name: async
-      description: Run the import asyncronously. Defaults to false.
-      type: boolean
-    - name: exec
-      description: Run the import when it is created. Defaults to false.
-      type: boolean
+  Contexts:
+    title: imports
+    type: array
+    description: List of import contexts.
+    items:
+      $ref: "#/definitions/Context"
+  Context:
+    title: context
+    type: object
+    description: An import context
+    properties:
+      id:
+        type: string
+        description: The import id
+      href:
+        type: string
+        description: URL to the import context endpoint
+      state: 
+        type: string
+        description: 'State of the import. One of: "INIT", "INIT_ERROR", "PENDING", "RUNNING", "COMPLETE"'
+      message:
+        type: string
+        description: The current context message, if any
+      archive:
+        type: boolean
+        description: Flag to control whether imported files (indirect) should be archived after import
+      targetWorkspace:
+        type: string
+        description: Target workspace of the import, if any.
+      targetStore:
+        $ref: "#/definitions/Store"
+      data:
+        $ref: "#/definitions/Data"
+      transforms:
+        $ref: "#/definitions/Transforms"
+      tasks:
+        $ref: "#/definitions/Tasks"
+  Tasks:
+    title: tasks
+    description: A list of tasks
+    type: array
+    items:
+      $ref: "#/definitions/Task"
+  Task:
+    title: task
+    type: object
+    description: An import task
+    properties:
+      id:
+        type: string
+        description: The task id
+      href:
+        type: string
+        description: URL to the task endpoint
+      state: 
+        type: string
+        description: 'State of the task. One of: "PENDING", "READY", "RUNNING", "NO_CRS", "NO_BOUNDS", "NO_FORMAT", "BAD_FORMAT", "ERROR", "CANCELED", "COMPLETE"'
+      updateMode:
+        type: string
+        description: 'Update mode of the task. One of: "CREATE", "REPLACE", "APPEND", "UPDATE"'
+      data:
+        $ref: "#/definitions/Data"
+      target:
+        $ref: "#/definitions/Store"
+      progress:
+        type: string
+        description: URL to the progress endpoint for this task
```

Comment:
None of the importer stuff goes through XStream, Importer uses its own JSON writer instead. Encoded as a string, but does follow URL syntax.